### PR TITLE
Fixed getting dessign issue while loading the page

### PR DESCRIPTION
**Pull request by amol2jcommerce**
### Description (*)

Getting dessign issue while loading the page.
### Fixed Issues (if relevant)

1. #20273: getting dessign issue while loading the page
2. Magento 2.3.0

### Manual testing scenarios (*)

1. admin Panel
2. Catalog--> Product
3. click edit link on created product

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-21609.instances.magento-community.engineering).
Admin access: [https://pr-21609.instances.magento-community.engineering/admin](https://pr-21609.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_actions-bar.less [hunk 1]
```diff
@@ -36,6 +36,17 @@
...
     }
+
+    .page-actions {
+        > button {
+            float: right;
+        }
+        > button#back {
+            float: left;
+            position: absolute;
+            right: 348px;
```
Comment (rogyar) [inline app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_actions-bar.less]: Please, avoid using hardcoded values. In that way, we introduce a workaround but not a solution. 
Try viewing the page with ~1005px screen width. The "back" button behaves better now but still "jumps"

**Review by rogyar — CHANGES_REQUESTED** [review]

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_actions-bar.less (hunk 1 above)
Comment (abrarpathan19) [reply app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_actions-bar.less]: @magento-engcom-team give me test instance

**Comment by abrarpathan19** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19, here is your [new Magento instance](https://pr-21609.instances.magento-community.engineering).
Admin access: [https://pr-21609.instances.magento-community.engineering/admin](https://pr-21609.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_actions-bar.less (hunk 1 above)
Comment (amol2jcommerce) [reply app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_actions-bar.less]: we use variable against hardcoded value. also now it's looking good for ~1005px screen width.
@rogyar  Please review.

**Comment by amol2jcommerce** [comment]
@rogyar please review.

**Comment by amol2jcommerce** [comment]
Hi @rogyar, @magento-engcom-team please review.

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-21609.instances.magento-community.engineering).
Admin access: [https://pr-21609.instances.magento-community.engineering/admin](https://pr-21609.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rogyar** [comment]
Hi @amol2jcommerce. The button is still jumping (below is a GIF, need to wait a little):

Backbutton

We need to consider another solution here (inherited positioning) instead of moving the element by pixels. Thank you.

**Comment by amol2jcommerce** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce, here is your [new Magento instance](https://pr-21609.instances.magento-community.engineering).
Admin access: [https://pr-21609.instances.magento-community.engineering/admin](https://pr-21609.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by amol2jcommerce** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce, here is your [Magento instance](https://i-21609-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21609-2-3-develop.instances.magento-community.engineering/admin](https://i-21609-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by amol2jcommerce** [comment]
Hi @rogyar, branch updated. please check once.

**Comment by rogyar** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-21609.instances.magento-community.engineering).
Admin access: [https://pr-21609.instances.magento-community.engineering/admin](https://pr-21609.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rogyar** [comment]
@amol2jcommerce now it's much better. Thank you

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-5031](https://jira.corp.magento.com/browse/ENGCOM-5031) has been created to process this Pull Request

**Comment by soleksii** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @stoleksiy, here is your [new Magento instance](https://pr-21609.instances.magento-community.engineering).
Admin access: [https://pr-21609.instances.magento-community.engineering/admin](https://pr-21609.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by soleksii** [comment]
Hi @amol2jcommerce ! I can not reproduce the issue #20273 .

**Result:**
vid

Could you please recheck on fresh 2.3-develop instance?!
Thanks!

**Comment by amol2jcommerce** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce, here is your [Magento instance](https://i-21609-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21609-2-3-develop.instances.magento-community.engineering/admin](https://i-21609-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by sidolov** [comment]
@amol2jcommerce , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**